app/views/IncomeSourceSummary.scala.html

```diff
@@ -84,32 +84,18 @@
 
     @if(model.rtiAvailable) {
         @if(model.isPension) {
```

Comment:
@anujakhare @vimalsuresh when it comes to testing this ticket you will need to test a pension user and an employment user